### ISSUE TITLE
Application Title

### ISSUE THREAD

**Issue by Niglb** [byteben/Win32App-Migration-Tool#1]
Not a huge issue,

Is there a switch or parameter to allow for Application Name instead of GUID? Also when I go to import into InTune is does not know the name either. Perhaps I am not doing this correctly. 

Love the module!! Keep it up!

**Comment by byteben** [comment]
Hey, thanks for the feedback. You pass the app name using the -AppName parameter. This Parameter also accepts wildcards e.g. 
`New-Win32App -SiteCode BB1 -ProviderMachineName "bb-cm1.byteben.com" -AppName "Microsoft*" -PackageApps`. 
All the details required to build the Win32App in Intune are saved in the %WorkingFolder%\Details\DeploymentTypes.csv
Did I understand correctly? Perhaps you can share a screenshot or the Main.log so I can help more. Thanks

**Comment by Niglb** [comment]
Ah the details folder is what I was missing - thank you! Let me try again.

**Comment by byteben** [comment]
No problem. Thanks for testing the tool! :)
